core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticEntity.java

```diff
@@ -14,10 +14,15 @@
     private DaoGeneticEntity() {
     }
     
+    // TODO: consider renaming this class to something something similar to DaoEntity 
+    // to reflect the fact that there are also entities that are "not genetic", like
+    // for example the new entity of type "TREATMENT".
+    // see: https://github.com/cBioPortal/cbioportal/pull/5460#discussion_r250148672
     public static enum EntityTypes
     {
         GENE,
         GENESET,
+        TREATMENT,
```

Comment:
Do we need TREATMENT type here?  Can it be replaced with GENERIC_ENTITY (I question the need for this enum, but thats a different discussion).

Reply:
No, this can also be renamed to GENERIC_ASSAY (I did). Please be aware that we did not removed all references to the concept of treatments from the existing code. This we envisioned to be future work associated with introduction of a generic _genetic_entity_ database structure.